dis_snek/api/http/http_requests/invites.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from dis_snek.client.const import MISSING, Absent
+from ..route import Route
+
+
+class InviteRequests:
```

Comment:
This file is never used, and you never added it to 
https://github.com/Discord-Snake-Pit/Dis-Snek/blob/e85c04abf42650958a2a750674650cccbc6456f1/dis_snek/api/http/http_client.py#L52